draft-ietf-tls-svcb-ech.md

```diff
@@ -96,7 +96,7 @@ A SVCB RRSet containing some RRs with "ech" and some without is vulnerable to a
 
 In an idealized deployment, ECH protects the SNI with an anonymity set consisting of all the ECH-enabled server domains supported by a given client-facing server. Accordingly, an attacker who can enumerate this set can always guess the encrypted SNI with probability 1/K, where K is the number of domains in the set. In practice, this probability may be increased via traffic analysis, popularity weighting, and other mechanisms.
 
-ECH ensures that TLS does not disclose the SNI, but the same information is also revealed in the DNS queries used to resolve the server's hostname.  To achieve the security goals from {{Section 10.1 of !ECH}}, clients for this specification SHOULD use an authenticated and encrypted connection to a DNS resolver that they trust to preserve the confidentiality of their queries and return authentic answers.
+ECH ensures that TLS does not disclose the SNI, but the same information is also revealed in the DNS queries used to resolve the server's hostname.  This specification does not conceal the server name from the DNS resolver.  If DNS messages are sent between the client and resolver without authenticated encryption, an attacker on this path can also learn the destination server name.
```

Comment:
s/is also/can also be/ ?

Reply:
I changed "is also revealed" to "is also present".

Reply:
Good catch.